ci-operator/step-registry/mirror-images/by-oc-adm-in-bastion/mirror-images-by-oc-adm-in-bastion-commands.sh

```diff
@@ -75,7 +75,7 @@ if ! whoami &> /dev/null; then
 fi
 
 SSH_PRIV_KEY_PATH=${CLUSTER_PROFILE_DIR}/ssh-privatekey
-BASTION_IP=$(<"${SHARED_DIR}/bastion_private_address")
+BASTION_IP=$(<"${SHARED_DIR}/bastion_public_address")
```

Comment:
I guess this change would break other install workflow, e.g: nutanix and vsphere. 

On nutanix and vsphere, the bastion host only has  private address. I think here need some condition for your baremetal case.

Reply:
This was not needed, changed back to `mirror-images-by-oc-adm`